packages/bpk-component-drawer/src/BpkDrawerContent.js

```diff
@@ -50,21 +50,21 @@ type Props = {
 
 const BpkDrawerContent = (props: Props) => {
   const {
+    children,
     className,
-    hideTitle,
+    closeLabel,
+    closeOnScrimClick,
+    closeText,
     contentClassName,
-    isDrawerShown,
-    onCloseAnimationComplete,
-    id,
     dialogRef,
-    title,
-    closeText,
-    onClose,
-    closeLabel,
-    children,
-    closeOnScrimClick, // unusued from withScrim scrim HOC
-    isIphone, // unusued from withScrim scrim HOC
-    isIpad, // unusued from withScrim scrim HOC
+    hideTitle,
+    id,
+    isDrawerShown,
+    isIpad,
+    isIphone,
+    onClose, // unusued from withScrim scrim HOC
```

Comment:
I think the reshuffle messed up the comments here. Also there is a typo in unusued

Reply:
Ah yes reshuffle doesn't move comments 🤦 Good spot with the mis-spell (not me but good catch from years ago)